pkg/apis/v2beta2/receiver_types.go

```diff
@@ -244,6 +244,41 @@ type PushoverReceiver struct {
 	TmplText *ConfigmapKeySelector `json:"tmplText,omitempty"`
 }
 
+// FeishuChatBot is the configuration of ChatBot
+type FeishuChatBot struct {
+	// The webhook of ChatBot which the message will send to.
+	Webhook *Credential `json:"webhook"`
+
+	// Custom keywords of ChatBot
+	Keywords []string `json:"keywords,omitempty"`
+
+	// Secret of ChatBot, you can get it after enabled signature verification of ChatBot.
```

Comment:
=>
	// Secret of ChatBot, you can get it after enabling signature verification of ChatBot.